specs/data_structures.md

```diff
@@ -75,6 +76,7 @@ Data Structures
 | `Amount`                    | `uint64`                   |
 | [`BlockID`](#blockid)       | [HashDigest](#hashdigest)  |
 | `FeeRate`                   | `uint64`                   |
+| `Graffiti`                  | `NamespaceID`              |
```

Comment:
> The burn tx includes a graffiti field to specify the burn reason.

Sounded more like a free text memo field btw. why not call the field "Namespace" instead of graffiti?

Reply:
So, my reasoning for having a separate graffiti type is that it's not 100% guaranteed that burning will be targeted to a particular namespace. It could be anything! Using namespace ID directly would introduce a tight coupling. If we decide to shrink the namespace ID later, then we'd have to change the field in the transaction from namespace ID to e.g. hash digest. With a separate graffiti type alias, we can just change the type being aliased.

Reply:
> With a separate graffiti type alias, we can just change the type being aliased.

which is effectively the same as changing the field (has the same implications at least).